sdk/azidentity/managed_identity_client.go

```diff
@@ -124,7 +124,10 @@ func (c *managedIdentityClient) createAccessToken(res *azcore.Response) (*azcore
 	}
 	// this is the case when expires_on is a time string
 	// this is the format of the string coming from the service
-	if expiresOn, err := time.Parse("01/02/2006 15:04:05 PM +00:00", value.ExpiresOn); err == nil { // the date string specified in the layout param of time.Parse cannot be changed, Golang expects whatever layout to always signify January 2, 2006 at 3:04 PM
+	if expiresOn, err := time.Parse("01/02/2006 15:04:05 PM +00:00", value.ExpiresOn); err == nil { // the date string specified is for Windows OS
```

Comment:
The month and day are zero-padded which I believe is incorrect.  Comment applies for below.

Reply:
Ok, I tested it and it works without the padding characters. Removed them. Done.